### PR TITLE
Fp2052 port lympne

### PR DESCRIPTION
**Pull request by nickine**
#### What does this PR do? (please provide any background)
Mock up SBP booking flow with Port Lympne branding

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
This will not be complete as it was only focusing on Availability/Extras/Payment therefore any consideration to modals has not been made.

#### Screenshots / Screencast
screen shot 2017-09-25 at 10 22 52
screen shot 2017-09-25 at 12 09 56

#### What gif best describes how you feel about this work?
https://i.imgur.com/XjimVTK.gifv

- [X] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [x] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/port-lympne/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@holidayextras/brand-port-lympne",
+  "version": "0.0.1",
+  "description": "Default",
```
Comment (jodiedoubleday) [inline brands/port-lympne/package.json]: pretty poor description

File: brands/port-lympne/src/less/hr.less [hunk 2]
```diff
@@ -0,0 +1,34 @@
+hr {
+  &.hr-brand {
+    background: transparent url('http://www.udopt.co.uk/wp-content/uploads/2012/11/Aspinall-Foundation-200x253.jpg') no-repeat 50%;
```
Comment (jodiedoubleday) [inline brands/port-lympne/src/less/hr.less]: Should add this in assets

File: brands/port-lympne/src/less/mixins/list_group.less [hunk 3]
```diff
@@ -0,0 +1,31 @@
+//
```
Comment (jodiedoubleday) [inline brands/port-lympne/src/less/mixins/list_group.less]: Interesting you added a new mixin. Is this necessary just for this brand? Not saying its not, but interested to hear your thoughts

Comment (nickine) [reply brands/port-lympne/src/less/mixins/list_group.less]: This comes with the rest of the Default brand which I used as the base for this project

File: brands/port-lympne/src/less/hr.less (hunk 2 above)
Comment (nickine) [reply brands/port-lympne/src/less/hr.less]: done

File: brands/port-lympne/package.json (hunk 1 above)
Comment (nickine) [reply brands/port-lympne/package.json]: you'll love the new one

**Pull request merged**